alf/environments/parallel_torch_environment.py

```
@@ -155,7 +153,14 @@ def _stack_time_steps(self, time_steps):
         """Given a list of TimeStep, combine to one with a batch dimension."""
         if self._flatten:
             return nest.fast_map_structure_flatten(
-                lambda *arrays: torch.stack(arrays), self._time_step_spec,
+                # TODO: right now ``self._time_step_spec`` is independenty defined
```

Let's throw an error before return, to make sure this is not called?

---

added